blockchain/blockdao/blockdao.go

```diff
@@ -56,6 +57,8 @@ type (
 	blockDAO struct {
 		blockStore   BlockDAO
 		indexers     []BlockIndexer
+		// add analyser indexer
+		analyserIndexers []BlockIndexer
```

Comment:
any reasons to split indexes into two parts?

Reply:
There are many analyserIndexers, and they will not block the chain server from starting, nor will they block each other. so analyserIndexers asynchronously catch up with the tipHeight
code is [here](https://github.com/iotexproject/iotex-core/pull/4352/files#diff-cab2e4d938c21357ad6f17be5bad1c5c1672f85dbc3ee8493769b48f04658debR189)